pkg/graph/schema.graphql

```diff
@@ -36,12 +36,20 @@ type Mutation {
   ): CreateAccessibilityRequestPayload
 }
 
+type System {
+  intakeId: UUID
+  lifecycleId: String
+  projectName: String
+  ownerName: String
+}
```

Comment:
A general piece of advice I've seen is that when you need to disambiguate a field by prepending it with a scope (like `projectName` or `ownerName`), it's better to use nested objects so that you have a place to put additional fields and extract types so that the schema is more descriptive:


```suggestion
type System {
  intakeId: UUID
  lifecycleId: String
  project: Project 
  owner: SystemOwner
}

type Project {
  name: String
}

type SystemOwner {
  name: String
```

Generally it seems like types that are specific to an association would be named as such, but in this example I'm assuming that `Project` is a top-level concept in the API. It-s likely not for us- this is just for illustration purposes.

I also think that we might want to make all of the fields on `System` required unless we're going to commit to either the system working without them or at the very least handling every case where we don't have that data.